File: python/1822/solution.py
class Solution(object):
    def arraySign(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        pro = 1
        for numeros in nums:
            pro *= numeros

        if pro > 0:
            return 1
        elif pro < 0:
            return -1
        return 0
    
        

File: java/1822/solution.java
class Solution {
    public int arraySign(int[] nums) {
        int n = nums.length;
        double pro = 1;
        for (int i =0; i < n; i++){
            pro *= nums[i];
        }

        if (pro>0){
            return 1;
        } else if (pro < 0){
            return -1;
        } 
        return 0;
    }
}